Gemfile

```diff
@@ -123,7 +123,7 @@ gem 'rolify'
 gem 'flipflop', '~> 2.3'
 gem 'lograge'
 
-gem 'hyrax-doi', git: 'https://github.com/samvera-labs/hyrax-doi.git', branch: 'play_nice_with_view_paths'
+gem 'hyrax-doi', git: 'https://github.com/samvera-labs/hyrax-doi.git', branch: 'main'
```

Comment:
Why what this branch reference changed?

Reply:
Of course! @ShanaLMoore the reference branch was changed due to the fact the play_nice branch has been deleted.